app/_how-tos/custom-deck-object-defaults.md

```diff
@@ -0,0 +1,153 @@
+---
+title: decK object defaults reference
+
+works_on:
+    - on-prem
+    - konnect
+
+tags:
+  - api-ops
+
+tools:
+  - deck
+
+related_resources:
+  - text: decK object defaults reference
+    url:  /deck/object-defaults
+
+tldr:
+  q: |
+    How do I set custom default values for Kong Gateway entities?
+  a: |
+    You can configure your own custom Kong Gateway defaults via decK to
+    enforce a set of standard values and avoid repetition in your configuration.
+---
+
+
+You can set custom configuration defaults for the following core
+{{site.base_gateway}} objects:
+- Service
+- Route
+- Upstream
+- Target
+
+Default values get applied to both new and existing objects. See the
+[order of precedence](#value-order-of-precedence) for more detail on how they
+get applied.
+
+You can choose to define custom default values for any subset of entity fields,
+or define all of them. decK still finds the default values using a
+combination of your defined fields and the object's schema, based on the
+order of precedence.
+
+decK supports setting custom object defaults both in self-managed
+{{site.base_gateway}} and with {{site.konnect_saas}}.
+
+{:.important}
+> **Important:** This feature has the following limitations:
+* Custom plugin object defaults are not supported.
+* If an existing property's default value changes in a future {{site.base_gateway}} release,
+decK has no way of knowing that this change has occurred, as its `defaults`
+configuration would overwrite the value in your environment.
+
+1. In your `kong.yaml` configuration file, add an `_info` section with
```

Comment:
@lena-larionova we changed this to be h2s instead of a list, each h2 (and the content after it) is treated as a step.